### ISSUE TITLE
__float__ method makes numpy/pandas forget currency

### ISSUE THREAD

**Issue by jonashaag** [django-money/django-money#347]
```
>>> pandas.DataFrame([moneyed.Money()]).sum()
0    0.00 XYZ
dtype: object

>>> pandas.DataFrame([djmoney.money.Money()]).sum()
0    0.0  # <-- currency missing
dtype: float64
```

This is due to implementing the `__float__` method:

```py
class MM(moneyed.Money):
    def __float__(self):
        return float(self.amount)
```

```
>>> pandas.DataFrame([MM()]).sum()
0    0.0  # <-- currency missing
dtype: float64
```

I'm not saying the `__float__` method should be removed/changed because I'm not sure of what use it is. Just wanted to leave a note about this here.

**Comment by benjaoming** [comment]
Hey!

A PR is totally welcome to fix this :) Maybe you can model the behavior of pandas in some test case (but without using pandas directly though) ?

**Comment by benjaoming** [comment]
This should be fixed in #391 and released as 0.13.3
